README.md

```diff
@@ -1,5 +1,17 @@
 # Little Esty Shop
 
+## Description of Project
```

Comment:
I went ahead and added the rough template for the README so we had it ready to go